packages/tools/client-logger/app-insights-logger/src/FluidAppInsightsLogger.ts

```diff
@@ -0,0 +1,53 @@
+import { ApplicationInsights, Snippet } from "@microsoft/applicationinsights-web";
+
+import { ITelemetryBaseEvent } from "@fluidframework/common-definitions";
+import { ITelemetryBufferedLogger } from "@fluidframework/test-driver-definitions";
```

Comment:
you don't need this interface. No one will every downcast to it, and use it, the class can just have a flush method

Reply:
switched to just using the base logger interface in commit 70aa9fd